collector/pg_long_running_transactions.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"context"
+
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const longRunningTransactionsSubsystem = "long_running_transactions"
+
+func init() {
+	registerCollector(longRunningTransactionsSubsystem, defaultDisabled, NewPGLongRunningTransactionsCollector)
+}
+
+type PGLongRunningTransactionsCollector struct {
+	log log.Logger
+}
+
+func NewPGLongRunningTransactionsCollector(config collectorConfig) (Collector, error) {
+	return &PGLongRunningTransactionsCollector{log: config.logger}, nil
+}
+
+var (
+	longRunningTransactionsCount = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, longRunningTransactionsSubsystem, "count"),
```

Comment:
`_count` is reserved for histogram/summary counters. We can just truncate the name here since the "unit" is transactions.

```suggestion
		prometheus.BuildFQName(namespace, longRunningTransactionsSubsystem, ""),
```